atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/paging/DelegatingColumnGetter.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.cassandra.paging;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.cassandra.thrift.ColumnOrSuperColumn;
+import org.apache.cassandra.thrift.KeySlice;
+
+import com.google.common.collect.Maps;
+
+public class DelegatingColumnGetter implements ColumnGetter {
```

Comment:
This is a confusing name. When you usually get DelegatingX, it has an underlying X which it uses.